scripts/county_adjacency_msa/index.ts

```diff
@@ -92,7 +126,7 @@ function main() {
   const data = parseAdjacencyFile(filePath);
   const adjacencyMap = parseCountyData(data);
 
-  const outputPath = path.join(__dirname, 'county_adjacency.json');
+  const outputPath = path.join(__dirname, 'county_adjacency_msa.json');
```

Comment:
Maybe it's a better idea to set the output path to be the final location of the file, that way we don't need to copy it over manually.